active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,335 @@
+
+![A prototype of five vertical dots with the second initially selected and the previous and next buttons appearing and disappearing based on user hover and the selected page](images/PipsPager_VisibleOnPointerOver.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VerticalPipsPager"
+  NumberOfPages="5"
+  Orientation=Vertical 
+  PreviousButtonVisibility=VisibleOnPointerOver 
+  NextButtonVisibility= VisibleOnPointerOver />
+```
+
+#### Number of pages displayed less than total
+
+If the number of pages of content is large and do not need to be navigated to at once,
+the MaxVisiblePips property can be set to limit the number of visible, interactable, pips.
+If the NumberOfPages is greater than the MaxVisiblePips then the pips will scroll so that the selected page is
+centered in the control. If the NumberOfPages is equal to or less than the MaxVisiblePips,
+no scrolling will occur and the number of pips shown will be the same as the NumberOfPages.
+
+By default, the number of visible pips is 5.
+
+![A prototype of twelve horizontal dots with the first initially selected and the dots scrolling when the subsequent pages are selected](images/PipsPager_MaxVisiblePips.gif)
+
+XAML
+```XAML
+<muxc:PipsPager x:Name="ScrollingPipsPager"
+  NumberOfPages="20"
+  MaxVisiblePips="10" />
+```
+
+#### Integrate PipsPager with a Collection control
+
+Commonly, a PipsPager is used in conjunction with a Collection control.
+The below example shows how to bind a PipsPager with a FlipView to provide
+an extra way to navigate through content and indicate its page.
+
+Note: To use the PipsPager as a page indicator **only**,
+set IsEnabled to false in the control to disable user interaction.
+
+![A prototype of five dots, the PipsPager, laid out horizontally underneath a FlipView showing a picture of a bicycle. The third dot in the PipsPager is selected indicated the 3rd page of content.](images/PipsPager_FlipViewExample.png)
+
+XAML
+``` XAML
+<StackPanel>
+  <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+      <FlipView.ItemTemplate>
+          <DataTemplate x:DataType="x:String">
+              <Image Source="{x:Bind Mode=OneWay}"/>
+          </DataTemplate>
+      </FlipView.ItemTemplate>
+  </FlipView>
+
+  <!-- The SelectedPageIndex is bound to the FlipView to keep the two in sync -->
+  <muxc:PipsPager x:Name="FlipViewPipsPager"
+      HorizontalAlignment="Center"
+      Margin="0, 10, 0, 0"
+      NumberOfPages="{x:Bind Pictures.Count}"
+      SelectedPageIndex="{x:Bind Path=Gallery.SelectedIndex, Mode=TwoWay}" />
+</StackPanel>
+```
+#### Pip and Button customization
+
+The navigational buttons and pips can be customized via the PreviousButtonStyle, NextButtonStyle, SelectedPipStyle, and DefaultPipStyle.
+As a note, setting the PreviousButtonStyle and NextButtonStyle can overwrite what is set in PreviousButtonVisibility and NextButtonVisibility.
```

Comment:
@oldnewthing you're correct, it's more clear this way.
It will not take precedence only in 1 case: if the **.....ButtonVisibility** is set to **Collapsed** then no matter what was in the .....ButtonStyle passed into the Pager, the Button will be collapsed at all times.

Reply:
Thanks, I addressed this in the latest update. I included the Visibility, Opacity, and IsEnabled properties as properties that will take precedence in the button Styles that are set in the visual state groups. @beervoley We can follow up on which exact properties take precedence offline.

Reply:
Updated once more to clarify after discussion offline with @beervoley:

> If the PreviousButtonStyle or NextButtonStyle sets the Visibility property on the navigational button, it will take precedence over PreviousButtonVisibility or NextButtonVisibility respectively unless they are set to the PipsPagerButtonVisibility value, "Collapsed". 
> If the PreviousButtonVisibility or NextButtonVisibility is set to "Collapsed", the styles set will not overwrite the Visibility of the button itself.